### PR TITLE
tools: remove hardcoding of bash hashbang

### PR DESCRIPTION
**Pull request by thomaseichinger**
For host systems like FreeBSD, bash binary is found in a different path.

### CODE REVIEW

**Comment by OlegHahm** [comment]
I would assume that for most of these scripts even any POSIX compliant shell would be enough, i.e. using sh instead of bash.

**Comment by A-Paul** [comment]
`#!env ....` doesn't work. (Debian GNU/Linux 8.2 (jessie), Fedora release 22 (Twenty Two)).
```
user@host:/tmp$ ./debug.sh 
bash: ./debug.sh: env: bad interpreter: No such file or directory
```

If you could search for the binary after "!", you would also be able to write `#!bash` without using env.

**Comment by Kijewski** [comment]
Related: http://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang

**Comment by thomaseichinger** [comment]
@A-Paul Could you verify if `/usr/bin/env` works for your tested distros?

**Comment by LudwigKnuepfer** [comment]
`/usr/bin/env` is what I would have put there in the first place.

**Comment by A-Paul** [comment]
> ... Could you verify if `/usr/bin/env` works for your tested distros?

@thomaseichinger yes it works. But I can't assure `env` to be in `/usr/bin/`  on every system. :-)

Also worth reading regarding the pros and cons of `#!/usr/bin/env ...`:
http://unix.stackexchange.com/questions/29608/why-is-it-better-to-use-usr-bin-env-name-instead-of-path-to-name-as-my

> I would assume that for most of these scripts even any POSIX compliant shell would be enough, i.e. using sh instead of bash.

I Agree. And `sh` is in `/bin` on every system as I know.

**Comment by OlegHahm** [comment]
> @thomaseichinger yes it works. But I can't assure env to be in /usr/bin/ on every system. :-)

Well, I guess using this shebang is still the best one can do and a BCP.

File: boards/x86-multiboot-common/dist/link [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!env bash
```
Comment (LudwigKnuepfer) [inline boards/x86-multiboot-common/dist/link]: missed one

**Comment by thomaseichinger** [comment]
@LudwigKnuepfer Thanks for the hint, fixed now.

**Comment by cgundogan** [comment]
Ready for squashing

**Comment by A-Paul** [comment]
@thomaseichinger. ACK when sqashed.

**Comment by cgundogan** [comment]
kicked travis again

**Comment by cgundogan** [comment]
Travis agrees - GO

**Pull request merged**